pytorch_toolkit/ote/tools/train.py

```diff
@@ -28,6 +32,9 @@ def main():
     arg_converter = build_arg_converter(modules['arg_converter'])
     train_args = arg_converter.convert_train_args(MODEL_TEMPLATE_FILENAME, ote_args)
 
+    config_transformers_engine = ConfigTransformersEngine(MODEL_TEMPLATE_FILENAME, modules.get("config_transformers"))
```

Comment:
As far as I remember we agreed on the fact that there should be two separate procedures: training and compression

Reply:
Please add a separate script

Reply:
Done